templates/includes/install-problems.hbs

```diff
@@ -2,28 +2,50 @@
+    <h3>Bootloop of any kind or stuck boot process</h3>
+    Don't panic. None of the commands we advised you to use can damage or brick your watch permanently.<br>
+    You can always go back to the fastboot bootloader menu, by using the <a href="https://asteroidos.org/wiki/useful-commands/#boot-to-fastboot-bootloader-menu">manual finger combos for your watch</a>.
+    <ul>
+      <li><h4>Verify that you used the correct image files</h4>Some devices like the Asus Zenwatch 2, the Fossil Gen4 models and the MTK6580 watches have more than one system image supplied, for different variants with slightly different hardware. Most watches have a graphical fastboot menu that shows the correct codename for the watch. Others need to be identified using the hardware differences noted above in the install section.</li>
+      <li><h4>Clean your watch and charger contacts</h4>Even in case the fastboot flash process succeeds with no errors, it can still silently fail due to connection problems caused by dirty contacts. Clean the pins and pads with isopropyl alcohol. Ideally using that old toothbrush you keep around for electronics cleaning or a microfiber cloth. Carefully sanding the contacts (e.g. with a nailfile) can help in case of obvious corrosion.</li>
+      <li><h4>Ensure a stable USB connection</h4>Put the watch and your computer on the same flat surface. Try to not move the cable connecting your computer and the watch during the whole flashing process. In case of worn or corroded contacts and pogo-pins that lost their springiness, it might be useful to attach the watch to its cradle or charging cable using duct tape or rubber bands.</li>
+      <li><h4>Repeat the flash process.</h4>Some users have reported needing up to five flashing attempts for a successful result.<br>There is a small chance that the image files were corrupted during download. We do not currently provide checksums for the files. However, if you download the files again under a different filename and then compare them with <code>sha256sum filename</code>, the checksums must match.</li>
+    </ul>
+    <h3>Boot gets stuck even after repeated reflash</h3>
+    Congratulations, you might have found a new issue for us to solve. While this should not happen with stable builds, the nightlies are advancing quickly, and are not tested on all watches. Since the watches install pages link to the nightlies currently, you likely used those. You can try again and <a href="https://release.asteroidos.org/">install a stable build when available</a>.<br>To help us solve the problem, you could try to collect logs. It is possible that the watch does not show a UI but is already accessible using <code>ssh ceres@192.168.2.15</code> or <code>adb shell</code> in the state the boot process stops. In case SSH connection works, try the following commands to write logs to local .txt files.
```

Comment:
I would rephrase `Since the watches install pages link to the nightlies currently, you likely used those.`, it sounds a bit strange to me.

Maybe something like:
```
You're likely using a nightly build since the install pages link to those currently.
```

Reply:
Great streamlining. Flow is much better like that.

Reply:
opted for `You are likely using a nightly build, as the installation pages currently link to it.`
After some back and forth translation in deepl.com